lib/locator.dart

```diff
@@ -10,7 +11,7 @@ GetIt locator = GetIt.I;
 void setUpLocator() {
   locator.registerLazySingleton(() => NavigationService());
   locator.registerLazySingleton(() => SharedPreferencesHelper());
-
+  locator.registerLazySingleton(() => NotificationWorkHelper());
```

Comment:
```suggestion
  locator.registerSingletonAsync<NotificationWorkHelper>(() async {
    final notificationWorkHelper = NotificationWorkHelper();
    await notificationWorkHelper.init();
    return notificationWorkHelper;
  })
```
Daria para fazer dessa forma, ai na main voce so pegaria ja inicializado
https://pub.dev/packages/get_it